<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>66</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_secret">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>66</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_secret">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="p_login">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="p_cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
